workspaces/ui-v2/src/components/navigation/TopNavigation.tsx

```diff
@@ -3,61 +3,79 @@ import { makeStyles } from '@material-ui/core/styles';
+  const changelogHistoryPage = useChangelogHistoryPage();
   const diffsPage = useDiffReviewPageLink();
+  const shouldRenderChangelogHistory =
+    process.env.REACT_APP_FF_SHOW_REVERT_COMMIT === 'true';
 
   const { AccessoryNavigation } = props;
   return (
-    <div className={classes.root} key="top-navigation">
-      <Container maxWidth={false} style={{ paddingLeft: 0, paddingRight: 0 }}>
-        <AppBar position="static" color="transparent" elevation={0}>
-          <Toolbar className={classes.toolbar}>
-            <div className={classes.stacked}>
-              <Typography
-                className={classes.title}
-                variant="subtitle2"
-                noWrap
-                component="span"
-              >
-                {apiName}
-              </Typography>
+    <div>
+      <div className={classes.root} key="top-navigation">
+        <Container maxWidth={false} style={{ paddingLeft: 0, paddingRight: 0 }}>
+          <AppBar position="static" color="transparent" elevation={0}>
+            <Toolbar className={classes.toolbar}>
+              <div className={classes.stacked}>
+                <Typography
+                  className={classes.title}
+                  variant="subtitle2"
+                  noWrap
+                  component="span"
+                >
+                  {apiName}
+                </Typography>
 
-              <NavButton
-                title="Docs"
-                to={documentationPage.linkTo()}
-                Icon={SubjectIcon}
-              />
-              {/*@aidan: this needs to change*/}
-              {appConfig.navigation.showDiff && (
                 <NavButton
-                  title="Diffs"
-                  to={diffsPage.linkTo()}
-                  Icon={ChangeHistoryIcon}
+                  title="Docs"
+                  to={documentationPage.linkTo()}
+                  Icon={SubjectIcon}
                 />
-              )}
-            </div>
-            <div className={classes.spacer} />
-            <div>{AccessoryNavigation && <AccessoryNavigation />}</div>
-          </Toolbar>
-        </AppBar>
-      </Container>
+                {shouldRenderChangelogHistory && (
+                  <NavButton
+                    title="Changelog"
+                    to={changelogHistoryPage.linkTo()}
+                    Icon={ListIcon}
+                  />
+                )}
+
+                {/*@aidan: this needs to change*/}
+                {appConfig.navigation.showDiff && (
+                  <NavButton
+                    title="Diffs"
+                    to={diffsPage.linkTo()}
+                    Icon={ChangeHistoryIcon}
+                  />
+                )}
+              </div>
+              <div className={classes.spacer} />
+              <div>{AccessoryNavigation && <AccessoryNavigation />}</div>
+            </Toolbar>
+          </AppBar>
+        </Container>
+      </div>
+      <div className={classes.spacing}></div>
```

Comment:
Added a spacing component such that the TopNavigation automatically spaces everything below the `position:fixed` top bar